contrib/scripts/dumpcerts.sh

```diff
@@ -149,10 +149,10 @@ for domain in $(jq -r '.DomainsCertificate.Certs[].Certificate.Domain' ${acmefil
 	echo "Extracting cert bundle for ${domain}"
 	cert=$(jq -e -r --arg domain "$domain" '.DomainsCertificate.Certs[].Certificate |
          	select (.Domain == $domain )| .Certificate' ${acmefile}) || bad_acme
-	echo "${cert}" | base64 --decode > "${cdir}/${domain}.crt"
+	echo "${cert}" | base64 -d > "${cdir}/${domain}.crt"
```

Comment:
Is `-d` supported in non-Alpine contexts as well?

If yes: cool :+1:

If no: shouldn't our script be smart enough to handle both option styles?

Reply:
The [man page](https://linux.die.net/man/1/base64) states both options. Ubuntu 10.04, Ubuntu 16.04, Centos 7 and Fedora 26 support the short-hand option.

Alpine is a bit special, because the installed binary for `base64` is actually the `busybox` binary. I guess that's the reason. I believe it is safe to use `-d` without checking for the environment.

Reply:
The base64 binary shipped with Mac OS X seems to deviate:

```$ base64 --help
Usage:	base64 [-hvD] [-b num] [-i in_file] [-o out_file]
  -h, --help     display this message
  -D, --decode   decodes input
  -b, --break    break encoded string into num character lines
  -i, --input    input file (default: "-" for stdin)
  -o, --output   output file (default: "-" for stdout)
```

:(

Reply:
That's sad... I'll have to check if the is another base64 binary for alpine.

The alternative would be to check if the base64 binary is actually a symlink to busybox and then revert to the -d option. If that's an acceptable solution, I would change my PR.

Reply:
How about just checking in the script whether [we run on Darwin](https://stackoverflow.com/questions/394230/detect-the-os-from-a-bash-script) and adjust the parameter accordingly?